tests/testthat/_snaps/tabPanel.md

```diff
@@ -142,6 +142,41 @@
         <div class="content-footer"></div>
       </div>
 
+---
+
+    Code
+      dropdown_active
+    Output
+      <div class="tabbable">
+        <ul class="nav nav-tabs" data-tabsetid="4785">
+          <li>
+            <a href="#tab-4785-1" data-toggle="tab" data-bs-toggle="tab" data-value="A">A</a>
+          </li>
+          <li>
+            <a href="#tab-4785-2" data-toggle="tab" data-bs-toggle="tab" data-value="B">
+              <i aria-label="github icon" class="fab fa-github fa-fw" role="presentation"></i>
+              B
+            </a>
+          </li>
+          <li class="dropdown active">
```

Comment:
Before the change in https://github.com/rstudio/bslib/pull/372, this `.dropdown` container was missing the `.active` class